src/main/kotlin/com/zaxxer/ping/IcmpPinger.kt

```diff
@@ -246,7 +252,11 @@ class IcmpPinger(private val responseHandler:PingResponseHandler) {
          val pingTarget = pendingPings.peek()
          if (sendIcmp(pingTarget, fd)) {
             pendingPings.take()
-            waitingTarget4Map[pingTarget.sequence] = pingTarget
+            if(pingTarget.isIPv4) {
```

Comment:
Whitespace after ``if`` please.  ``if`` is not a function or method. 😄

Reply:
Sorry for formatting errors.